va/va_dec_av1.h

```diff
@@ -38,6 +38,26 @@
 extern "C" {
 #endif
 
+/** Attribute value for VAConfigAttribValDecAV1Features.
```

Comment:
VAConfigAttribDecAV1Features -> VAConfigAttribDecAV1Features.

Reply:
sorry, should be VAConfigAttribValDecAV1Features -> VAConfigAttribDecAV1Features